src/core/t-vector.c

```diff
@@ -707,6 +707,8 @@ void Mold_Vector(const REBVAL *value, REB_MOLD *mold, REBOOL molded)
         // remove final space
         //
         SET_SERIES_LEN(mold->series, SER_LEN(mold->series) - 1);
+        Append_Codepoint_Raw(mold->series, '\0');
+        SET_SERIES_LEN(mold->series, SER_LEN(mold->series) - 1);
```

Comment:
There's actually a routine for arrays called TERM_ARRAY_LEN which does both these steps... I think we need TERM_SEQUENCE_LEN.

*(Sequence is currently the name for non-array series... if it were just TERM_SERIES_LEN it would have to check if it was an array or not as the first thing it did.)*

Thanks for verifying that this was the issue; I'll add TERM_SEQUENCE_LEN and do a search and replace on patterns like this with it!